MBTI Quiz/js/script.js

```diff
@@ -157,31 +111,27 @@ function ScrolltoResults(){
       q5class[i].style.display='block';
     }
 
-  introverted = extraverted = sensory = intuitive = feeling = thinking = judging = prospecting = 0;
 
-if(document.getElementById('introverted').checked){
-  introverted += 100;
-}
-if(document.getElementById('extraverted').checked){
-  extraverted += 100;
-}
-if(document.getElementById('sensory').checked){
-  sensory += 100;
-}
-if(document.getElementById('intuitive').checked){
-  intuitive += 100;
-}
-if(document.getElementById('feeling').checked){
-  feeling += 100;
-}
-if(document.getElementById('thinking').checked){
-  thinking += 100;
-}
-if(document.getElementById('judging').checked){
-  judging += 100;
-}
-if(document.getElementById('prospecting').checked){
-  prospecting += 100;
+//introverted, extraverted, sensory, intuitive, feeling, thinking, judging, prospecting = 0;
+
+var preferences = [
+	{name: 'introverted', score: 0},
+  {name: 'extraverted', score: 0},
+  {name: 'intuitive', score: 0},
+  {name: 'sensory', score: 0},
+  {name: 'feeling', score: 0},
+  {name: 'thinking', score: 0},
+  {name: 'judging', score: 0},
+  {name: 'prospecting', score: 0}
+];
+
+//loop through ids on question 1 and add appropriote scores
+for (var i = 0; i < preferences.length; i++)
+{
+  if(document.getElementById(preferences[i].name).checked)
+  {
```

Comment:
Opening bracket should be same line as if statement and same for above for loop

Reply:
fixed